pyro/poutine/trace.py

```diff
@@ -50,49 +100,84 @@ def __init__(self, *args, **kwargs):
         super(Trace, self).__init__(*args, **kwargs)
 
     def __del__(self):
-        # Work around cyclic reference bugs in networkx.DiGraph
-        # See https://github.com/uber/pyro/issues/798
+        """
+        Work around cyclic reference bugs in ``networkx.DiGraph``
+        See ``https://github.com/uber/pyro/issues/798``
+        """
         self._graph.__dict__.clear()
 
     @property
     def nodes(self):
+        """
+        Identical to ``networkx.DiGraph.nodes``
```

Comment:
If you link to sphinx docs in our conf.py
```diff
diff --git a/docs/source/conf.py b/docs/source/conf.py
index 73da0c17..65175989 100644
--- a/docs/source/conf.py
+++ b/docs/source/conf.py
@@ -175,6 +175,7 @@ texinfo_documents = [
 intersphinx_mapping = {
     'python': ('https://docs.python.org/3/', None),
     'torch': ('http://pytorch.org/docs/master/', None),
+    'networkx': ('https://networkx.github.io/documentation/stable/', None),
 }
 
 # document class constructors (__init__ methods):
```
then you could promote these to links via
```
"""
Identical to :attr:`networkx.DiGraph.nodes`
"""
```
(I think `:attr:` is correct here, others may need `:meth:`)